cfep-10.md

```diff
@@ -0,0 +1,59 @@
+
+<table>
+<tr><td> Title </td><td> Feedstock Statuses </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Christopher J. "CJ" Wright &lt;cjwright4242@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 29, 2019</td></tr>
+<tr><td> Discussion </td><td> link to the PR where the CFEP is being discussed, NA is circulated initially </td></tr>
+<tr><td> Implementation </td><td> link to the PR for the implementation, NA if not availble </td></tr>
+</table>
+
+## Abstract
+
+This document proposes the creation of four new statuses for feedstocks that denote the	
+standing of the feedstock and transistions between the standings.
+These statuses will help us triage feedstocks and identify which feedstocks need
+help, maintainers, or archiving.
```

Comment:
Should explicitly state that this CFEP does not "implement" these statuses in terms of saying how they should be surfaced, tracked, etc.